pkg/formats/influx/influx.go

```diff
@@ -66,7 +66,7 @@ func (d *InfluxData) String() string {
 	i := 0
 	for k, v := range d.Fields {
 		if ev, ok := d.Tags[k]; ok { // There's an enum here, use this vs the int value.
-			fields[i] = k + "=" + ev.(string)
+			fields[i] = k + "=\"" + influxEscapeField(ev.(string)) + "\""
```

Comment:
you could write this as

```
fields[i] = fmt.Sprintf("%s=%q", k, influxEscapeField(ev.(string)))
```
Relying on %q to properly quote the value. 

Also, will ev ever not be a string?

Reply:
`%q` will wrap it in `"` and also apply escaping to make the result a valid Go string literal. That would do the right thing with internal `"` and `\` (good), but I think it would also turn newlines into `\n` and probably some other special characters. This probably doesn't matter in this code path since these are only enum names coming from snmp profile yaml files, but I'd rather keep it explicit.